app/models/info_request.rb

```diff
@@ -83,6 +83,8 @@ class InfoRequest < ActiveRecord::Base
   has_tag_string
 
   scope :visible, Prominence::VisibleQuery.new
+  scope :embargoed, Prominence::EmbargoedQuery.new
+  scope :unembargoed, Prominence::UnembargoedQuery.new
```

Comment:
Or just `:public` - we have a `public?` method so it makes sense to me to make that a matching scope too